scripts/build/preloadSchemas.ts

```diff
@@ -0,0 +1,59 @@
+/*!
```

Comment:
instead of this we can shallow-clone the whole https://github.com/awslabs/goformation repo (`master` branch). That can't possibly be blocked...